CLI/commands/sto_manager.js

```diff
@@ -314,22 +315,74 @@ function addressesConfigUSDTieredSTO(usdTokenRaise) {
   });
   if (addresses.reserveWallet == "") addresses.reserveWallet = Issuer.address;
 
+  let listOfAddress;
+
   if (usdTokenRaise) {
-    addresses.usdToken = readlineSync.question('Enter the address of the USD Token or stable coin (' + usdToken.options.address + '): ', {
+    addresses.usdToken = readlineSync.question('Enter the address (or many addresses that you want separated by comma) of the USD Token or stable coin (' + usdToken.options.address + '): ', {
       limit: function (input) {
-        return web3.utils.isAddress(input);
+        listOfAddress = input.split(',');
+        let response = true
+        listOfAddress.forEach((addr) => {
+          if (!web3.utils.isAddress(addr)) {
+            response = false
+          }
+        })
+        return response
       },
       limitMessage: "Must be a valid address",
       defaultInput: usdToken.options.address
     });
-    if (addresses.usdToken == "") addresses.usdToken = usdToken.options.address;
+    if (addresses.usdToken == "") {
+      listOfAddress = [usdToken.options.address]
```

Comment:
Why do you have two list of addresses? You can use 'addresses.usdToken' everywhere

Reply:
nop, I need to keep both vars. If I remove listOfAddress, processArray brokes.